src/modules/rust.rs

```diff
@@ -707,7 +715,7 @@ version = "12"
         let rustc_nightly = "rustc 1.34.0-nightly (b139669f3 2019-04-10)";
         assert_eq!(
             format_rustc_version(rustc_nightly, config.version_format),
-            Some("v1.34.0-nightly".to_string())
+            Some("nightly-2019-04-10".to_string())

```

Comment:
I think this is not correct because the actual rustup toolchain version and commit date are consistently off by one. (https://github.com/rust-lang/rust/issues/51533)